src/post/header.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PublishedDate from './publishedDate'
+import UserId from './userId'
+
+class Header extends React.Component{
+
+    render() {
+        return( 
+            <>
+                <PublishedDate/>
```

Comment:
This could have been implemented as a single component.

Reply:
Also, add a space between `PublishedDate` and `/>`